applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py

```diff
@@ -216,13 +236,27 @@ def Initialize(self):
         # Check if everything is assigned correctly
         self._check()
       
-        if self._is_printing_rank():
-            KratosMultiphysics.Logger.PrintInfo("ParticleMPMSolver","Solver is initialized correctly.")
+        self.print_on_rank_zero("::[ParticleMPMSolver]:: ","Solver is initialized correctly.")
 
-    def Solve(self):
-        (self.solver).Solve()
+    def AdvanceInTime(self, current_time):
+        dt = self.ComputeDeltaTime()
+        new_time = current_time + dt
+        self.model_part1.ProcessInfo[KratosMultiphysics.STEP] += 1
+        self.model_part1.CloneTimeStep(new_time)
+
+        return new_time
+
+    def ComputeDeltaTime(self):
+        return self.settings["time_stepping"]["time_step"].GetDouble()
 
+    def SolveSolutionStep(self):
+        (self.solver).Solve()
 
+    def Solve(self):
```

Comment:
please do not add this, the proper calls including the deprecation-msg are already in the BaseClass